Pod/Classes/Cards/Card.swift

```diff
@@ -40,7 +40,7 @@ public struct Card {
      
      - returns: A card with the provided parameters.
      */
-    public static func create(number: String, cvc: String, expiry: String) throws -> Card {
+    public static func create(_ number: String, cvc: String, expiry: String) throws -> Card {
```

Comment:
Isn't it more descriptive and in line with the new Swift 3 style to keep this as is and call it like this: `Card.create(number: ..., cvc: ..., expiry: ...)`?